gen3/bin/dbbackup.sh

```diff
@@ -222,28 +224,289 @@ copy_to_aurora() {
     gen3 job run psql-db-copy-aurora SOURCE_NAMESPACE "$1"
 }
 
-# main function to determine whether dump, restore, or create service account
+# Function to perform encrypted backup
+encrypt_backup() {
+    gen3 job run psql-db-backup-encrypt
+}
+
+# Function to set up cronjob for encrypted backup
+setup_cronjob() {
+    gen3 job cron psql-db-backup-encrypt "15 1 * * *"
+}
+
+# Create policy for Mountpoint for Amazon S3 CSI driver
+create_s3_csi_policy() {
```

Comment:
Same thing for all these other functions related to csi driver. These should go into a `kube-setup-s3-csi` or similar script, and then this script can call that other script, we need to make sure the other script is idempotent as well.